src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -349,23 +355,15 @@ default int maxConnections() {
 	 * {@link Connection}. Further connections will be pending acquisition depending on
 	 * pendingAcquireTime. The maximum number of connections is for the connections in a single
 	 * connection pool, where a connection pool corresponds to a concrete remote host.
+	 * The configuration can be either default for all connection pools, or per a connection pool.
```

Comment:
I'd say "The configuration can be either `global` for all connection pools or `be tuned for each individual pools, per host`."

the per pool/host aspect can be combined at will, right? (ie if I configure for 1 host in particular, all other hosts would get treated by the global config?)

Reply:
> the per pool/host aspect can be combined at will, right? (ie if I configure for 1 host in particular, all other hosts would get treated by the global config?)

yes